frontend/src/components/program/ScheduleEntries.vue

```diff
@@ -56,15 +60,23 @@ export default {
   },
   computed: {
     scheduleEntries() {
-      // TODO for SideBar, add filtering for the current day, now that the API supports it
-      return this.period().scheduleEntries()
+      if (this.date) {
```

Comment:
Müssten diese ifs nicht mit this.day arbeiten? this.date ist glaube ich immer undefined

Reply:
🙃

Reply:
einä vo denä momentä he.. merci 😅 auso zum klarsteuä, dä PR macht stand 399e1ef _nüt_ wöus immer i `false` branch geit was drzuä füärt das genau dr glich code usgfüärt wird wi vorem PR.

Reply:
Confirmed in browser dev tools that it now fetches data with before and after guards instead of the whole period.